packages/framework-provider-azure-infrastructure/src/infrastructure/synth/terraform-containers.ts

```diff
@@ -4,6 +4,8 @@ import { toTerraformName } from '../helper/utils'
 import { BoosterConfig } from '@boostercloud/framework-types'
 import { eventsStoreAttributes } from '@boostercloud/framework-provider-azure'
 
+const MAX_CONTAINER_THROUGHPUT = 2000
```

Comment:
Maybe we should move this into a file `constants.ts`?

Reply:
Ouch, merged before reading your comment @NickSeagull , sorry! I will move it in following PRs